samples/powerpoint/99-preview-apis/get-set-shapes.yaml

```diff
@@ -0,0 +1,222 @@
+order: 2
+id: powerpoint-get-set-shapes
+name: 'Get, set, load, and save shapes'
+description: Get and set one or more selected shapes. Load and save one or more shapes.
+host: POWERPOINT
+api_set: {}
+script:
+    content: |
+        $("#getSelectedShapes").click(() => tryCatch(getSelectedShapes));
+        $("#setSelectedShapes").click(() => tryCatch(setSelectedShapes));
+        $("#changeFill").click(() => tryCatch(changeFill));
+        $("#saveShapeSelection").click(() => tryCatch(saveShapeSelection));
+        $("#loadShapeSelection").click(() => tryCatch(loadShapeSelection));
+        $("#createShapes").click(() => tryCatch(createShapes));
+        $("#arrangeSelected").click(() => tryCatch(arrangeSelected));
+
+        async function getSelectedShapes() {
+          await PowerPoint.run(async (context) => {
```

Comment:
include brief description of each function -- I recommend inserting the description immediately before `await PowerPoint.run...` statement